test/regress/regress-37.txt

```diff
@@ -0,0 +1,136 @@
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 -2 3 4 5 6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 -3 4 5 6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 -4 5 6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 -5 6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 -6 7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 -7 8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 -8 9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 -9 10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 -10 11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 -11 12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 -12 13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 -13 14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 -14 15))
+(assert_return (invoke "i8x16") (v128.const i8x16 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 -15))
+
+(assert_return (invoke "i16x8") (v128.const i16x8 -1 1 2 3 4 5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 -1 2 3 4 5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 -2 3 4 5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 2 -3 4 5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 2 3 -4 5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 2 3 4 -5 6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 2 3 4 5 -6 7))
+(assert_return (invoke "i16x8") (v128.const i16x8 0 1 2 3 4 5 6 -7))
+
+(assert_return (invoke "i32x4") (v128.const i32x4 -1 1 2 3))
+(assert_return (invoke "i32x4") (v128.const i32x4 0 -1 2 3))
+(assert_return (invoke "i32x4") (v128.const i32x4 0 1 -2 3))
+(assert_return (invoke "i32x4") (v128.const i32x4 0 1 2 -3))
+
+(assert_return (invoke "i64x2") (v128.const i64x2 -1 1))
+(assert_return (invoke "i64x2") (v128.const i64x2 0 -1))
+
+(assert_return (invoke "f32x4") (v128.const f32x4 -1 1 2 3))
+(assert_return (invoke "f32x4") (v128.const f32x4 0 -1 2 3))
+(assert_return (invoke "f32x4") (v128.const f32x4 0 1 -2 3))
+(assert_return (invoke "f32x4") (v128.const f32x4 0 1 2 -3))
+
+(assert_return (invoke "f64x2") (v128.const f64x2 -1 1))
+(assert_return (invoke "f64x2") (v128.const f64x2 0 -1))
+(;; STDOUT ;;;
+out/test/regress/regress-37.txt:21: mismatch in lane 0 of result 0 of assert_return: expected i8:255, got i8:0
+out/test/regress/regress-37.txt:21: mismatch in result 0 of assert_return: expected v128 i8:255i8:1i8:2i8:3i8:4i8:5i8:6i8:7i8:8i8:9i8:10i8:11i8:12i8:13i8:14i8:15, got v128 i32x4:0x03020100 0x07060504 0x0b0a0908 0x0f0e0d0c
```

Comment:
Agree that it's confusing, it comes from [TypedValueToString](https://github.com/WebAssembly/wabt/blob/main/src/interp/interp-util.cc#L40) formatting a v128 value in this way, regardless of the expected lane type. This could be easily fixed, but as this PR doesn't touch that code, and it could cause a bunch of updates in other tests whose output would change, I think it's better to do this in a separate PR.

Reply:
Also it's showing the asserted values, not the "got" ones (note that first `i8:255`). The formatting of all the `i8:`s here is also quite hard to read, but again not touched by this PR.